test/unit/modules/signatures.js

```diff
@@ -14,66 +14,200 @@
 
 'use strict';
 
+const application = require('../../common/application');
+const transactionTypes = require('../../../helpers/transaction_types');
+const ApiError = require('../../../helpers/api_error');
+
 /* eslint-disable mocha/no-pending-tests */
 describe('signatures', () => {
+	let library;
+
+	before(done => {
+		application.init(
+			{ sandbox: { name: 'lisk_test_modules_signatures' } },
+			(err, scope) => {
+				library = scope;
+				done(err);
+			}
+		);
+	});
+
 	describe('isLoaded', () => {
-		it('should return true if modules exists');
+		let revert;
 
-		it('should return true if modules does not exist');
+		afterEach(() => {
+			return revert();
+		});
+
+		it('should return true if modules exists', () => {
+			revert = library.rewiredModules.signatures.__set__('modules', {
+				accounts: library.modules.accounts,
+				transactions: library.modules.transactions,
+				transport: library.modules.transport,
+			});
+			return expect(library.modules.signatures.isLoaded()).to.be.true;
+		});
+
+		it('should return true if modules does not exist', () => {
+			revert = library.rewiredModules.signatures.__set__('modules', undefined);
+			return expect(library.modules.signatures.isLoaded()).to.be.false;
+		});
 	});
 
 	describe('onBind', () => {
+		let privateModules;
+		let signatureLogicSpy;
+
+		before(() => {
+			privateModules = library.rewiredModules.signatures.__get__('modules');
+			signatureLogicSpy = sinonSandbox.spy(
+				library.rewiredModules.signatures.__get__('__private').assetTypes[
+					transactionTypes.SIGNATURE
+				],
+				'bind'
+			);
+			return library.modules.signatures.onBind(library.modules);
+		});
+
+		after(() => {
+			return signatureLogicSpy.restore();
+		});
+
```

Comment:
I think moving `signatureLogicSpy` definition  `assetTypes` suite would be better since they were only being used there ?

Reply:
Makes total sense. Done